src/libraries/src.proj

```diff
@@ -49,7 +49,7 @@
              Properties="$(TraversalGlobalProperties)" />
   </Target>
 
-  <Import Condition="'$(BuildTargetFramework)' == '$(NetCoreAppCurrent)'"
+  <Import Condition="'$(BuildTargetFramework)' == '$(NetCoreAppCurrent)' or '$(BuildAllConfigurations)' == 'true'"
```

Comment:
why? I don't think we should link in all configurations build we already do it in all the netcoreapp verticals.